es6-persistence/src/main/java/com/netflix/conductor/es6/config/ElasticSearchConditions.java

```diff
@@ -27,13 +27,13 @@ private ElasticSearchConditions() {
         }
 
         @SuppressWarnings("unused")
-        @ConditionalOnProperty(prefix = "workflow", name = "indexing.enabled", havingValue = "true", matchIfMissing = true)
+        @ConditionalOnProperty(name = "conductor.indexing.enabled", havingValue = "true", matchIfMissing = true)
         static class enabledIndexing {
 
         }
 
         @SuppressWarnings("unused")
```

Comment:
Considering the fact that es5-persistence is moved out and the default is not ES6. Is this property needed? Can we constrain ES6 enablement on just this `conductor.indexing.enabled`?

Reply:
We can remove this property, but historically, we have always added the support for the next elasticsearch version to co-exist with currently supported version. With version 1.x es2 was default (es5 shaded support), with version 2.x es5 is default (es6 shaded support). I believe we will continue this pattern with es7 in v3.x. The existence of this property makes it easy to add this module in the future. 
If it feels too hand-wavy for the future, I can remove this.